tests/sim/gridworld/test_state.py

```diff
@@ -27,6 +28,28 @@ def test_position_state():
     assert grid[2, 0] == {'agent2': agents['agent2']}
 
 
+def test_position_state_no_overlap_at_reset():
```

Comment:
This test would be a lot more meaningful if we had a larger grid with N cells and N agents some with and some without initial positions, and we show that each agent has its own position and that each grid cell only has one agent on it. We can actually have multiple IP agents on the same cell, so we'll include that.